pkg/hwinfo/hwinfo_nonwindows.go

```diff
@@ -48,6 +101,23 @@ func fetchInventory() (map[string]interface{}, error) {
 		res["displays.list"] = displays
 	}
 
+	cpus, err := listCPUs()
+	errorCollector.Add(err)
+	if len(cpus) > 0 {
+		encodedCpus := make(map[string]interface{})
+
+		for i := range cpus {
```

Comment:
This new loop has a different abstraction level than the current function implementation. I think it should be moved inside listCPUs() func or to a new function that translates `cpus` to `encodedCpus`

Reply:
done